React/Modules/RCTAsyncLocalStorage.m

```diff
@@ -66,11 +79,16 @@ static void RCTAppendError(NSDictionary *error, NSMutableArray<NSDictionary *> *
 {
   static NSString *storageDirectory = nil;
   static dispatch_once_t onceToken;
+
+  NSSearchPathDirectory searchDir = NSDocumentDirectory;
+  if(storageLocation == ApplicationSupport)
```

Comment:
please follow coding style from the project, see: https://github.com/facebook/react-native/blob/master/React/Base/RCTBridge.m#L67-L69 for an example of an if statement with one line, and notice the space between `if` and `(`